apps/test/solutions/maze/hidden_start_blocks.js

```diff
@@ -0,0 +1,49 @@
block></value></block></next></block></next></block><block type="when_run" deletable="false" movable="false"><next><block type="controls_repeat"><title name="TIMES">2</title><statement name="DO"><block type="maze_moveForward"><next><block type="procedures_callnoreturn" inline="false"><mutation name="get 3 nectar"><arg name="direction"/></mutation><value name="ARG0"><block type="variables_get"><title name="VAR">left</title></block></value><next><block type="maze_moveForward"><next><block type="procedures_callnoreturn" inline="false"><mutation name="get 3 nectar"><arg name="direction"/></mutation><value name="ARG0"><block type="variables_get"><title name="VAR">right</title></block></value></block></next></block></next></block></next></block></statement></block></next></block><block type="procedures_defnoreturn" uservisible="false"><mutation><arg name="direction"/></mutation><title name="NAME">get 3 nectar</title><statement name="STACK"><block type="controls_if" inline="false" uservisible="false"><mutation else="1"/><value name="IF0"><block type="logic_compare" inline="true" movable="false" uservisible="false"><title name="OP">EQ</title><value name="A"><block type="variables_get" movable="false" uservisible="false"><title name="VAR">direction</title></block></value><value name="B"><block type="variables_get" uservisible="false"><title name="VAR">left</title></block></value></block></value><statement name="DO0"><block type="maze_turn" uservisible="false"><title name="DIR">turnLeft</title></block></statement><statement name="ELSE"><block type="maze_turn" uservisible="false"><title name="DIR">turnRight</title></block></statement><next><block type="maze_move" uservisible="false"><title name="DIR">moveForward</title><next><block type="maze_move" uservisible="false"><title name="DIR">moveForward</title><next><block type="controls_repeat" uservisible="false"><title name="TIMES">3</title><statement name="DO"><block type="maze_nectar" uservisible="false"/></statement><next><block type="maze_move" uservisible="false"><title name="DIR">moveBackward</title><next><block type="maze_move" uservisible="false"><title name="DIR">moveBackward</title><next><block type="controls_if" inline="false" uservisible="false"><mutation else="1"/><value name="IF0"><block type="logic_compare" inline="true" movable="false" uservisible="false"><title name="OP">EQ</title><value name="A"><block type="variables_get" movable="false" uservisible="false"><title name="VAR">direction</title></block></value><value name="B"><block type="variables_get" uservisible="false"><title name="VAR">left</title></block></value></block></value><statement name="DO0"><block type="maze_turn" uservisible="false"><title name="DIR">turnRight</title></block></statement><statement name="ELSE"><block type="maze_turn" uservisible="false"><title name="DIR">turnLeft</title></block></statement></block></next></block></next></block></next></block></next></block></next></block></next></block></statement></block></xml>'
```

Comment:
Might be worth a comment about what's interesting about this xml (I think it's that y is -5 for our first when_run)